### PR TITLE
Unify deprecation messages

### PR DESCRIPTION
**Pull request by maarcingebala**
This PR unifies all deprecation messages. From now on we want to provide dates of when a field will be removed from the schema, instead of particular versions of Saleor, as new versions aren't always released regularly.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

File: saleor/graphql/product/schema.py [hunk 1]
```diff
@@ -199,9 +199,8 @@ class ProductQueries(graphene.ObjectType):
         stock_availability=graphene.Argument(
             StockAvailability,
             description=(
-                "Filter products by stock availability."
-                "DEPRECATED: Will be removed in Saleor 2.11, "
-                "use the `filter` field instead."
+                "Filter products by stock availability. Use the `filter` field "
```
Comment (fowczarek) [inline saleor/graphql/product/schema.py:202]:
```suggestion
                "[Deprecated] Filter products by stock availability. Use the `filter` field "
```

:policeman:

File: saleor/graphql/product/types/products.py [hunk 2]
```diff
@@ -166,20 +166,23 @@ class ProductVariant(CountableDjangoObjectType):
         required=True,
         description="Quantity of a product in the store's possession, "
         "including the allocated stock that is waiting for shipment.",
-        deprecation_reason="This field will be removed in Saleor 2.11. "
-        "Use the stock field instead.",
+        deprecation_reason=(
+            "Use the stock field instead. This field will be removed after 2020-07-31"
```
Comment (IKarbowiak) [inline saleor/graphql/product/types/products.py:170]: Lack of dot at the end 👮‍♀️

File: saleor/graphql/product/types/products.py [hunk 3]
```diff
@@ -166,20 +166,23 @@ class ProductVariant(CountableDjangoObjectType):
...
+        ),
     )
     quantity_allocated = graphene.Int(
         required=False,
         description="Quantity allocated for orders",
-        deprecation_reason="This field will be removed in Saleor 2.11. "
-        "Use the stock field instead.",
+        deprecation_reason=(
+            "Use the stock field instead. This field will be removed after 2020-07-31"
```
Comment (IKarbowiak) [inline saleor/graphql/product/types/products.py:177]: Lack of dot at the end 👮‍♀️

File: saleor/graphql/product/types/products.py [hunk 4]
```diff
@@ -166,20 +166,23 @@ class ProductVariant(CountableDjangoObjectType):
...
+        ),
     )
     stock_quantity = graphene.Int(
         required=True,
         description="Quantity of a product available for sale.",
-        deprecation_reason="This field will be removed in Saleor 2.11. "
-        "Use the stock field instead.",
+        deprecation_reason=(
+            "Use the stock field instead. This field will be removed after 2020-07-31"
```
Comment (IKarbowiak) [inline saleor/graphql/product/types/products.py:184]: The same here 👮‍♀️

File: saleor/graphql/product/types/products.py [hunk 5]
```diff
@@ -197,8 +200,9 @@ class ProductVariant(CountableDjangoObjectType):
     )
     is_available = graphene.Boolean(
         description="Whether the variant is in stock and visible or not.",
-        deprecation_reason="This field will be removed in Saleor 2.11. "
-        "Use the stock field instead.",
+        deprecation_reason=(
+            "Use the stock field instead. This field will be removed after 2020-07-31"
```
Comment (IKarbowiak) [inline saleor/graphql/product/types/products.py:204]: And here 👮‍♀️

File: saleor/graphql/product/types/products.py [hunk 6]
```diff
@@ -385,7 +389,7 @@ class Product(CountableDjangoObjectType):
     url = graphene.String(
         description="The storefront URL for the product.",
         required=True,
-        deprecation_reason="DEPRECATED: Will be removed in Saleor 2.11.",
+        deprecation_reason="This field will be removed after 2020-07-31",
```
Comment (IKarbowiak) [inline saleor/graphql/product/types/products.py:392]: . 👮‍♀️

File: saleor/graphql/product/types/products.py [hunk 7]
```diff
@@ -760,7 +764,7 @@ class Category(CountableDjangoObjectType):
     )
     url = graphene.String(
         description="The storefront's URL for the category.",
-        deprecation_reason="DEPRECATED: Will be removed in Saleor 2.11.",
+        deprecation_reason="This field will be removed after 2020-07-31",
```
Comment (IKarbowiak) [inline saleor/graphql/product/types/products.py:767]: . 👮‍♀️

**Comment by maarcingebala** [comment]
@IKarbowiak @fowczarek Fixed 👮‍♀️👮‍♂️

**Pull request merged**